tests/commands.sh

```diff
@@ -126,6 +126,7 @@ TS_CMD_SETARCH=${TS_CMD_SETARCH-"${ts_commandsdir}setarch"}
 TS_CMD_SETPGID=${TS_CMD_SETPGID-"${ts_commandsdir}setpgid"}
 TS_CMD_SETPRIV=${TS_CMD_SETPRIV-"${ts_commandsdir}setpriv"}
 TS_CMD_SETSID=${TS_CMD_SETSID-"${ts_commandsdir}setsid"}
+TS_CMD_SU=${TS_CMD_SU-"${ts_commandsdir}su"}
```

Comment:
Any reason, the tests are not part of the introducing commit?

Reply:
The feature has been there for a long time. The proper would be to introduce the test before the current change (rewrite) :-)